crates/runc/src/lib.rs

```diff
@@ -749,25 +789,26 @@ mod tests {
 #[cfg(test)]
 #[cfg(all(target_os = "linux", feature = "async"))]
 mod tests {
+    use std::sync::Arc;
+
     use super::io::{InheritedStdIo, PipedStdIo};
     use super::*;
-    use std::sync::Arc;
 
-    fn ok_client() -> Runc {
+    fn ok_client() -> Runc<DefaultExecutor> {
```

Comment:
Let's keep `Runc` with different APIs depending on `async` toggle (see comment above)

Reply:
Maybe we can not make it,  as refer to the answer of the comment above.